Shared/Common/Extensions/UIViewController+.swift

```diff
@@ -42,6 +42,13 @@ protocol SpinnerAlertView {
 extension UIViewController: StatusAlertView {
     func displayTemporaryAlert(_ message: String, timeout: TimeInterval) {
         if let temporaryAlertView = Bundle.main.loadNibNamed("StatusAlert", owner: self)?.first as? StatusAlert {
+            
+            for view in self.view.subviews {
+                if view is StatusAlert {
+                    self.animateAlertOut(view, delay: timeout)
```

Comment:
Is this blocking? I suspect that there may still be some overlap of alert display within the `timeout` time period. Otherwise I think this approach is fine; curious what the UX folks think of the behavior here.